modules/ossm-mt-namespaces.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-mt-ossm.adoc
+
+[id="ossm-mt-namespaces_{context}"]
+= Configuring namespaces in multi-tenant installations
+
+A multi-tenant control plane installation only affects namespaces configured as part of the {ProductShortName}. You must specify the namespaces associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same namespace as the `ServiceMeshControlPlane` resource and name it `default`.
```

Comment:
Is it worth adding a warning somewhere stating that, if CNI is enabled, manual injection will work but that the pod will be unable to talk to the control plane if it's not in the SMMR?

Reply:
I think that's a pretty necessary piece of information for the user. It may belong in multiple places.

Reply:
I've added a note explaining this here, created a CNI module, and added that note to the CNI module as well.